.storybook/main.js

```diff
@@ -46,19 +47,21 @@ const config = {
     // You can change the configuration based on that.
     // 'PRODUCTION' is used when building the static version of storybook.
 
-    config.module.rules.push({
-      test: /\.mjs$/,
-      type: 'javascript/auto'
-    }, {
-      test: /\.js$/,
-      exclude: /node_modules/,
-      loader: 'babel-loader',
-      options: {
-        presets: [['@babel/preset-env', {
-          modules: 'commonjs'
-        }]]
-      }
-    }, {
+    config.module.rules.push(
+    // {
```

Comment:
Just double checking this commenting is fully intentional?

Reply:
Yeah for now at least. There's going to be a good bit of experimentation with the config over the next while to get it all properly dialled in so just commented out for now so it's handy.

Reply:
With the recent move to Storybook v7 a lot of the underlying webpack + babel config has changed, for example it now automatically inherits the project's babel config instead of us having to maintain a separate one specifically for Storybook. Similar with webpack, the defaults have changed and are now what I'd consider a lot more sensible, although I'm still having some issues with Carbon Charts (specifically the SASS build portion which is working fine in Code Sandbox…), might grab an hour of your time later in the week to try figure it out.